docs/developers/trace-xmlupload.md

```diff
@@ -0,0 +1,31 @@
+# Trace xmlupload
+
+## Preparation
+
+- update the dependencies in the virtual environment: `poetry install`
+- activate the virtual environment: `poetry shell`
```

Comment:
maybe we should also mention that a stack has to be running, it's obvious if you know dsp-tools well but but since this is a step-by-step guide, it feels like it should be here too.